app/src/main/java/com/framgia/soundclound/screen/local/LocalFragment.java

```diff
@@ -8,6 +8,8 @@
 import android.view.ViewGroup;
 
 import com.framgia.soundclound.R;
+import com.framgia.soundclound.data.source.TrackRepository;
```

Comment:
ERROR: Unused import: com.framgia.soundclound.data.source.TrackRepository. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports